requirements.txt

```diff
@@ -59,7 +59,6 @@ pycsw==2.6.1
 SQLAlchemy==2.0.19 # required by PyCSW
 Shapely==1.8.5.post1
 mercantile==1.2.1
-geoip2==4.7.0
```

Comment:
@mattiagiupponi we don't want to drop geoip2 functionality for the moment. Only the automatic update fo the DB